subprojects/core/src/testFixtures/groovy/org/gradle/api/tasks/AbstractCopyTaskContractTest.groovy

```diff
@@ -31,8 +30,8 @@ abstract class AbstractCopyTaskContractTest extends AbstractConventionTaskTest {
     @Override
     abstract AbstractCopyTask getTask()
 
-    @Unroll
-    def rootLevelFileCopyDetailsIsDslEnhanced() {
+    def "root level FileCopyDetails are DSL-enhanced"() {
+        expect:
```

Comment:
The assertions happen synchronously, right? (otherwise the execution will finish with success even in case of an error)

Reply:
You're absolutely right. That test didn't actually do anything... I added another commit.

Reply:
👍 I've checked and executed the tests locally. They fail only if I purposely modify the assertion statements. The PR is good to go.